src/__swaps__/screens/Swap/components/SwapOutputAsset.tsx

```diff
@@ -73,26 +75,28 @@ function SwapOutputAmount() {
   }, [navigate]);
 
   return (
-    <GestureHandlerV1Button
-      disableButtonPressWrapper
-      onPressStartWorklet={() => {
-        'worklet';
-        if (outputQuotesAreDisabled.value) {
-          runOnJS(handleTapWhileDisabled)();
-        } else {
-          focusedInput.value = 'outputAmount';
-        }
-      }}
-    >
-      <MaskedView maskElement={<FadeMask fadeEdgeInset={2} fadeWidth={8} height={36} side="right" />} style={styles.inputTextMask}>
-        <AnimatedText ellipsizeMode="clip" numberOfLines={1} size="30pt" style={SwapTextStyles.outputAmountTextStyle} weight="bold">
-          {SwapInputController.formattedOutputAmount}
-        </AnimatedText>
-        <Animated.View style={[styles.caretContainer, SwapTextStyles.outputCaretStyle]}>
-          <Box as={Animated.View} borderRadius={1} style={[styles.caret, AnimatedSwapStyles.assetToBuyCaretStyle]} />
-        </Animated.View>
-      </MaskedView>
-    </GestureHandlerV1Button>
+    <CopyPasteMenu onCopy={() => Clipboard.setString(SwapInputController.formattedOutputAmount.value)}>
```

Comment:
added just the copy option for output, iirc some swaps don't let you edit output, could add a paste and hide in this case

Reply:
we have a derivedValue in swap-provider called `outputQuotesAreDisabled` that you can use to disable pasting.